apps/system/test/marionette/software_home_file_open_error_test.js

```diff
@@ -48,34 +48,69 @@ marionette('Software Home Button - File Open Error', function() {
     system.gotoBrowser(url);
 
     // Save the file.
-    actions.longPress(client.helper.waitForElement('a'), 1).perform();
+    longPress(client.helper.waitForElement('a'));
     client.switchToFrame();
     system.appContextMenuSaveLink.click();
 
     // Tap on the toaster to open the download.
     // We could also open this from settings or the utility tray if needed.
-    var toasterTitle;
-    client.waitFor(function() {
-      toasterTitle = client.helper.waitForElement(
-        '#notification-toaster.displayed .toaster-title');
-      return toasterTitle.text().indexOf('Download complete') !== -1;
-    });
-    toasterTitle.tap();
+    getToasterTitle().tap();
 
-    function rect(el) {
-      return el.getBoundingClientRect();
-    }
+    var dialogHeight = getDialogHeight();
```

Comment:
DialogHeight is pretty generic. Since we are abstracting away the which dialog into a funciton, let's name the function something like `getDownloadDialogHeight` or something.